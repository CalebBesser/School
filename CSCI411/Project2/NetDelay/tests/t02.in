16
addEdge 0 3 11
addEdge 1 2 13
addEdge 2 3 8
addEdge 2 4 19
addEdge 4 6 12
addEdge 5 6 16
addEdge 5 11 24
addEdge 6 9 14
addEdge 7 8 35
addEdge 7 9 20
addEdge 7 10 29
addEdge 11 13 12
addEdge 11 14 5
addEdge 12 13 18
addEdge 14 15 7
printGraphW
findNetDelay
