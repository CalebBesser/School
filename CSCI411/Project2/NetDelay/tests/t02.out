Graph is:
0: 3/11	
1: 2/13	
2: 1/13	3/8	4/19	
3: 0/11	2/8	
4: 2/19	6/12	
5: 6/16	11/24	
6: 4/12	5/16	9/14	
7: 8/35	9/20	10/29	
8: 7/35	
9: 6/14	7/20	
10: 7/29	
11: 5/24	13/12	14/5	
12: 13/18	
13: 11/12	12/18	
14: 11/5	15/7	
15: 14/7	

The longest path is:
8 7 9 6 5 11 13 12 
The length of the longest path is 139.
